views/default/input/number.php

```diff
@@ -0,0 +1,23 @@
+<?php
+/**
+ * Elgg number input
+ * Displays a number input field
+ *
+ * @package Elgg
+ * @subpackage Core
+ *
+ * @uses $vars['class'] Additional CSS class
+ */
+
+$vars['class'] = (array) elgg_extract('class', $vars, []);
```

Comment:
How to proceed? It seems that the PR that adds elgg_extract_class() has not been merged yet. If I change elgg_extract() to elgg_extract_class() and this PR gets merged before the other, it will cause an error when using the view.